packages/cli/register/src/ir-to-fdr-converter/convertTypeShape.ts

```diff
@@ -113,6 +113,11 @@ export function convertTypeReference(
                 },
                 literal: (literal) => {
                     return Ir.types.Literal._visit(literal, {
+                        boolean: (booleanLiteral) =>
+                            // TODO: Refactor this when the FernRegsitry API supports boolean literals.
+                            FernRegistry.api.v1.register.TypeReference.literal(
+                                FernRegistry.api.v1.register.LiteralType.stringLiteral(booleanLiteral.toString())
+                            ),
```

Comment:
Here's the `TODO` for the FernRegistry - for now we just convert the boolean into a string, but we'll need to update this so that we actually capture it as a boolean literal.

Reply:
Discussed offline - made the change in the FernRegistry, but we'll follow-up to migrate to the latest SDK in a follow-up.